content/about/faq/_index.md

```diff
@@ -6,30 +6,53 @@ type = "faq"
+ - Last but not least is the RP factor, which is perhaps the most difficult one. RP is still a critical part of making a species unique, but at the same time we recognize that main SS14 servers will be low RP, and we are not sure if we would actually be able to to provide this.
 
-### Q: Is There a Patreon? Can I donate?
 
-A: [Yes.](https://www.patreon.com/spacestation14) We started a Patreon in order to pay for productivity software, server hosting, work commissions, and more! In addition to providing rewards, we will also regularly post a financial statement that shows how our income is being used in the game's development. We want to give our patrons full trust and confidence, and we believe that being 100% transparent about how the money is being used is best.
+ Due to these factors we feel like round-start playable (i.e. selectable from the character creator) will likely never happen. Areas like antagonist roles (e.g. vox raiders) are less problematic here, so they may happen, but the bar is still very high. And again, we still aim to make it easy for forks to add species if they want to.
+</details>
 
-### Q: Where can I play SS14?
+### Q: Will SS14 support multiple codebases/servers like SS13?
+It already does! You can checkout our [hosting guide](https://hackmd.io/@ss14/for-hosting) to get started.
 
-A: Spacestation 14 is currently in a loosly defined "pre-alpha" stage. You can download the client right now and play on our servers, but it's mainly for testing at the moment. A fair estimate of when the game will be 'complete' isn't possible, though we will be releasing the game for free on Steam around fall of 2020 as we continue to work on it.
+Note: If you're just looking to host a private server to mess around in solo play or test stuff, it's much less hassle to just [build the game yourself](https://hackmd.io/@ss14/docs/getting-set-up) which will compile a local client/server for you.
+
+### Q: Can I use SS14's engine to make other games?
+Yes, we even have templates for both [singleplayer](https://github.com/space-wizards/RobustToolboxTemplateSingleplayer) and [multiplayer](https://github.com/space-wizards/RobustToolboxTemplate) to get you started.
+
+### Q: Is There a Patreon? Can I donate?
+[Yes.](https://www.patreon.com/spacestation14) We started a Patreon in order to pay for productivity software, server hosting, work commissions, and more! In addition to providing rewards, we will also regularly post a financial statement that shows how our income is being used in the game's development. We want to give our patrons full trust and confidence, and we believe that being 100% transparent about how the money is being used is best.
```

Comment:
Replace "work commissions" with "art commissions" maybe? Since that's more accurate.
Also maybe link the finances spreadsheet here, since we already have that